Ivan sprint3

xaaniyaro opened the pull request:
- Ahora si la cuenta loggeada es Admin, la lista de eventos añade un botón "eliminar", funciona como se espera.
- Ahora si se da click en una zona fuera de los botones, se cierra el menú de seleccionar modo de juego.

xaaniyaro committed:
Salir menú Jugar y borrar evento

Ahora si el usuario da click en cualquier lugar de la pantalla que no sean los dos botones de los modos de juego, desaparece el menú.
Si el usuario es admin le aparece un botón de "borrar", si le da click se borra el evento de la base de datos y se desaparece de la interfaz

xaaniyaro committed:
Cambios en News.unity

Cambiado el layout de la vista de eventos

Noe9704 commented:
+ No deja borrar todos los anuncios, en el anuncio final marca un error

xaaniyaro committed:
Cambio turno por equipo y correcciones

Ahora los turnos son equipos, y se les asignó un color, toda sección involucrada fue actualizada.
Ahora se limpian los inputs de crear usuario.

napo-lazo commented:
- No se cargan las imágenes de los equipos en el menú principal
- Al borrar un evento se están borrando todos los demás también

xaaniyaro committed:
Fixed remove event

Se le asigna un id generado aleatoreamente para borrar eventos consistentemente

xaaniyaro committed:
Merge branch 'master' into IvanSprint3

xaaniyaro committed:
Updated CreateEvent and merged

xaaniyaro committed:
Fixed delete event and UserCreation

Se agregó más rango para la función que genera números random para asignar IDs.
Se borró una línea de código de la escena "UserCreation" que causaba un error.
Se agregó un cuadro de texto en la pantalla de eventos para checar si no se está creando un evento vacío.
